src/Pelagos/Entity/Account.php

```diff
@@ -317,6 +329,71 @@ public function getPasswordEntity()
         return $this->password;
     }
 
+    /**
+     * Returns the LoginAttempts Collection for this Account.
+     *
+     * @return LoginAttempts The LoginAttempts entity attached to this Account.
```

Comment:
return value is Collection of loginAttempts, we usually specify it as Collection